packages/imperative/__tests__/config/cmd/init/init.handler.test.ts

```diff
@@ -111,8 +111,9 @@ describe("Configuration Initialization command handler", () => {
         expect(setSchemaSpy).toHaveBeenCalledWith(expectedSchemaObjectNoBase);
 
         expect(promptWithTimeoutSpy).toHaveBeenCalledTimes(1);
+        // Prompting for secure property
         // tslint:disable-next-line: no-magic-numbers
-        expect(promptWithTimeoutSpy).toHaveBeenCalledWith(expect.stringContaining("blank to skip:"), true, 900); // Prompting for secure property
+        expect(promptWithTimeoutSpy).toHaveBeenCalledWith(expect.stringContaining("blank to skip:"), true, undefined);
```

Comment:
"// tslint:disable-next-line: no-magic-numbers" no longer needed - this also applies to other lines below in the file

Reply:
good point - in hindsight I'm not sure I get the point of why we had any of these in here.  I see `4` is used in some of the tests, and I don't know the significance of `4` - so, it's a magic number.  Likewise with other rule exclusions.

I'll try to change all of these